spec/Section 5 -- Validation.md

```diff
@@ -432,7 +432,6 @@ SameResponseShape(fieldA, fieldB) :
   * If {typeA} or {typeB} is Scalar or Enum.
     * If {typeA} and {typeB} are the same type return true, otherwise return
       false.
-  * If {typeA} or {typeB} is not a composite type, return false.
```

Comment:
You're right, but I think it's still worth clarifying that point in the algorithm.

How about instead replacing this step with:

* Assert: {typeA} and {typeB} are both composite types.

Reply:
Works for me. Updated.